hid-ghlive/src/hid-ghlive.c

```diff
@@ -41,121 +41,28 @@ static const char ghl_ps4_magic_data[] = {
 struct ghlive_sc {
 	struct hid_device *hdev;
 	unsigned long quirks;
-	int device_id;
+	int device_id; /* What is this for? */
```

Comment:
This is something I wanna talk to you about. is this int device_id used for anything? Doesn't seem to have a reason to be.

Reply:
> This is something I wanna talk to you about. is this `int device_id` used for anything? Doesn't seem to have a reason to be.

Erased the variable and tested. works fine. might have been an artefact from previous iterations?

Reply:
Yup. Clearly a leftover. Good catch!